Woosmap/geofencing-core-android-sdk
# Remove runtime permissions from library manifest

## Issue
closes https://github.com/Woosmap/geofencing-enterprise-android-sdk/issues/101

## Describe your changes
1. Removed all runtime permissions from the library manifest file to the sample app manifest file.

## How to test
1. Compile the SDK using `./gradlew assemblerelease`
2. Renamed the resulting AAR file extension to ZIP.
3. Extract the zip, and open the `AndroidManifest.xml`, and check if any runtime permissions are listed in the manifest.

## Checklist:



## Comments
